bookkeeper-server/src/test/java/org/apache/bookkeeper/client/GenericEnsemblePlacementPolicyTest.java

```diff
@@ -132,9 +132,9 @@ public void testReplaceBookie() throws Exception {
                     lh.addEntry(value);
                 }
             }
-            assertEquals(2, customMetadataOnNewEnsembleStack.size());
```

Comment:
@sijie  for instance I have a custom policy, which implements only old methods like the one in:
LegacyCustomDefaultEnsemblePlacementPolicyTest.java 

this is a real example:
https://github.com/diennea/herddb/blob/master/herddb-core/src/main/java/herddb/cluster/PreferLocalBookiePlacementPolicy.java

this example does not even extends DefaultEnsamblePlacementPolicy, so my current patch will not work :-(